tests/artifacts/openebs-ci.go

```diff
@@ -58,12 +58,13 @@ const (
 type LabelSelector string
 
 const (
-	MayaAPIServerLabelSelector           LabelSelector = "name=maya-apiserver"
-	OpenEBSProvisionerLabelSelector      LabelSelector = "name=openebs-provisioner"
-	OpenEBSSnapshotOperatorLabelSelector LabelSelector = "name=openebs-snapshot-operator"
-	OpenEBSAdmissionServerLabelSelector  LabelSelector = "app=admission-webhook"
-	OpenEBSNDMLabelSelector              LabelSelector = "name=openebs-ndm"
-	OpenEBSCStorPoolLabelSelector        LabelSelector = "app=cstor-pool"
+	MayaAPIServerLabelSelector             LabelSelector = "name=maya-apiserver"
+	OpenEBSProvisionerLabelSelector        LabelSelector = "name=openebs-provisioner"
+	OpenEBSLocalPVProvisionerLabelSelector LabelSelector = "name=openebs-localpv-provisioner"
+	OpenEBSSnapshotOperatorLabelSelector   LabelSelector = "name=openebs-snapshot-operator"
+	OpenEBSAdmissionServerLabelSelector    LabelSelector = "app=admission-webhook"
```

Comment:
The latest yaml in openebs/master has changed to name=admission-webhook. cc: @prateekpandey14

Reply:
We have the `app=admission-webhook` label present .. we have reverted this change, as its getting used by the webhook service.